### PR TITLE
Added Opinionated Guides

### PR DESCRIPTION
**Pull request by VegaDeftwing**
OpGuides is an open source book on all things technical seeking contributors

### CODE REVIEW

File: _data/projects/OpinionatedGuides.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+desc: a one-stop guide to computers, linux, and electronics.
+site: https://github.com/opinionatedguide/opinionatedguide.github.io-src
+tags:
+- awesome
+- documentation
+- python
+- bash
+upforgrabs:
+  name: hacktoberfest
+  link: https://github.com/opinionatedguide/opinionatedguide.github.io-src/issues/2
```
Comment (shiftkey) [inline _data/projects/OpinionatedGuides.yml]: This link should go to the list of available issues, not just to a specific issue

```suggestion
  link: https://github.com/opinionatedguide/opinionatedguide.github.io-src/labels/Hacktoberfest
```

File: _data/projects/OpinionatedGuides.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: Opinionated Guides
+desc: a one-stop guide to computers, linux, and electronics.
+site: https://github.com/opinionatedguide/opinionatedguide.github.io-src
+tags:
+- awesome
+- documentation
+- python
+- bash
+upforgrabs:
+  name: hacktoberfest
```
Comment (shiftkey) [inline _data/projects/OpinionatedGuides.yml]: This needs to be an exact match of the label in use: 

```suggestion
    name: Hacktoberfest
```

File: _data/projects/OpinionatedGuides.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+name: Opinionated Guides
+desc: a one-stop guide to computers, linux, and electronics.
```
Comment (shiftkey) [inline _data/projects/OpinionatedGuides.yml]: Minor change as this description will be listed on the site:

```suggestion
desc: A one-stop guide to computers, linux, and electronics.
```

File: _data/projects/OpinionatedGuides.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+site: https://github.com/opinionatedguide/opinionatedguide.github.io-src
```
Comment (shiftkey) [inline _data/projects/OpinionatedGuides.yml]: This link doesn't need to be the code, and could be to the site itself, e.g. `https://opinionatedguide.github.io/`

**Review by shiftkey — CHANGES_REQUESTED** [review]
@VegaDeftwing thanks for submitting!

I added some recommended changes to point to the correct label, and some suggestions where a different value might be more relevant.

File: _data/projects/OpinionatedGuides.yml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+name: Opinionated Guides
+desc: A one-stop guide to computers, linux, and electronics.
+site: https://github.com/opinionatedguide/opinionatedguide.github.io-src
+tags:
+- awesome
+- documentation
+- python
+- bash
+upforgrabs:
+    name: Hacktoberfest
```
Comment (shiftkey) [inline _data/projects/OpinionatedGuides.yml]: Not sure how this happened, but looks like I added in whitespace unnecessarily, triggering that error

```suggestion
  name: Hacktoberfest
```

**Comment by VegaDeftwing** [comment]
Removed the whitespace and changed the site link, I think it's good to go. Thank you so much and sorry for all the minor problems there!

File: _data/projects/OpinionatedGuides.yml (hunk 5 above)
Comment (VegaDeftwing) [reply _data/projects/OpinionatedGuides.yml]: fixed

File: _data/projects/OpinionatedGuides.yml [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+desc: A one-stop guide to computers, linux, and electronics.
+site: https://github.com/opinionatedguide/opinionatedguide.github.io-src
+tags:
+- awesome
+- documentation
+- python
+- bash
+upforgrabs:
+  name: Hacktoberfest
+  link: https://opinionatedguide.github.io
```
Comment (shiftkey) [inline _data/projects/OpinionatedGuides.yml]: This link was correct before:

```suggestion
  link: https://github.com/opinionatedguide/opinionatedguide.github.io-src/labels/Hacktoberfest
```

Comment (VegaDeftwing) [reply _data/projects/OpinionatedGuides.yml]: ahh, whoops, I changed link instead of site. Sorry, about that. Fixed

**Review by shiftkey — APPROVED** [review]

**Comment by shiftkey** [comment]
@VegaDeftwing thanks!

**Pull request merged**